tests/dynamic_api_test.py

```diff
@@ -1,1786 +0,0 @@
-# Copyright 2018 The JAX Authors.
```

Comment:
I wouldn't delete this test, this is absolutely something we want to make work at some point.

Some of it should even work without iree, e.g., the tests that build jaxprs.

Reply:
Reverted and removed mentions of iree.

Reply:
Should be okay now.